web/frontend/src/components/remote-control-cards.svelte

```diff
@@ -1,71 +1,82 @@
 <script lang="ts">
-
-import { type Credentials } from '@viamrobotics/rpc';
-import { commonApi } from '@viamrobotics/sdk';
-import { resourceNameToString, filterWithStatus, filterSubtype } from '@/lib/resource';
-import { useRobotClient } from '@/hooks/robot-client';
-import Arm from './arm/index.svelte';
-import AudioInput from './audio-input/index.svelte';
-import Base from './base/index.svelte';
-import Board from './board/index.svelte';
-import CamerasList from './camera/index.svelte';
-import OperationsSessions from './operations-sessions/index.svelte';
-import DoCommand from './do-command/index.svelte';
-import Encoder from './encoder/index.svelte';
-import Gantry from './gantry/index.svelte';
-import Gripper from './gripper/index.svelte';
-import Gamepad from './gamepad/index.svelte';
-import InputController from './input-controller/index.svelte';
-import Motor from './motor/index.svelte';
-import MovementSensor from './movement-sensor/index.svelte';
-import Navigation from './navigation/index.svelte';
-import Servo from './servo/index.svelte';
-import Sensors from './sensors/index.svelte';
-import Slam from './slam/index.svelte';
-import Client from '@/lib/components/robot-client.svelte';
-
-const { resources, components, services, statuses, sensorNames } = useRobotClient();
-
-export let host: string;
-export let bakedAuth: { authEntity?: string; creds?: Credentials; } | undefined = {};
-export let supportedAuthTypes: string[] | undefined = [];
-export let webrtcEnabled: boolean;
-export let signalingAddress: string;
-
-const resourceStatusByName = (resource: commonApi.ResourceName.AsObject) => {
-  return $statuses[resourceNameToString(resource)];
-};
-
-// TODO (APP-146): replace these with constants
-$: filteredWebGamepads = $components.filter((component) => {
-  const remSplit = component.name.split(':');
-  return (
-    component.subtype === 'input_controller' &&
-    Boolean(component.name) &&
-    remSplit[remSplit.length - 1] === 'WebGamepad'
-  );
-});
-
-/*
- * TODO (APP-146): replace these with constants
- * filters out WebGamepad
- */
-$: filteredInputControllerList = $components.filter((component) => {
-  const remSplit = component.name.split(':');
-  return (
-    component.subtype === 'input_controller' &&
-    Boolean(component.name) &&
-    remSplit[remSplit.length - 1] !== 'WebGamepad' && resourceStatusByName(component)
-  );
-});
-
-const getStatus = (statusMap: Record<string, unknown>, resource: commonApi.ResourceName.AsObject) => {
-  const key = resourceNameToString(resource);
-  // todo(mp) Find a way to fix this type error
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  return key ? statusMap[key] as any : undefined;
-};
-
+  import { type Credentials } from '@viamrobotics/rpc';
```

Comment:
Would it be possible to restore the original formatting? It's hard to figure out what changed in this script

Reply:
do you know of any good way? there wasn't a commit with the prettier and linting changes, I was just regularly applying it while I was making changes. I can comment on the parts that need review, if that helps

Reply:
I'd recommend making another PR first where you either format these files or add our shared prettier config: https://github.com/viamrobotics/js-config